### DISCUSSION TITLE
Initial Setup up - 500 error

### DISCUSSION THREAD

**Discussion by xq1xq1xq1** [Art-of-WiFi/UniFi-API-client#208] [General]
Unifi Controller version: 8.07
Debian: 11

I have cloned the site to Debian 11 and installed it per README

I copied the `config.php` file and modified it reflect my unifi controller installation using the user and password that for logging into the web portal and the associated URL

I commented out the composer autoloader line in the `list_sites.php` file and am using this for my test

`// require_once 'vendor/autoload.php';`

When I try to access the `list_sites.php` file I get this error:

`[19/Dec/2023:15:28:21 -0600] "GET /examples/list_sites.php HTTP/1.1" 500 185 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:120.0) Gecko/20100101 Firefox/120.0"`

I enabled DEBUG in my modified config.php but did not get any additional feedback

I assume I am missing something basic or this is not compatible with my version of controller

I did not find any posts with respect to 500 errors on the forum

Ideas on how to troubleshoot?

**Comment by malle-pietje** [comment]
Your web server error log will tell you more. 500 is a server error

**Reply by xq1xq1xq1** [reply]
The issue was the the with the wording of the installation section of the main doc

`Use [Composer](https://github.com/Art-of-WiFi/UniFi-API-client#composer), [Git](https://github.com/Art-of-WiFi/UniFi-API-client#git) or simply [Download the Release](https://github.com/Art-of-WiFi/UniFi-API-client#download-the-release) to install the API client class.`

it seemed like Composer was one option but rather it is a prerequisite 

all good now

This can be closed

**Comment by xq1xq1xq1** [comment]
Once I ran composer it worked

**Comment by malle-pietje** [comment]
I’ll double check the instructions. Thing is, without composer you have to have an understanding of the relative file paths for the class and the config file.

**Comment by powej2** [comment]
I'm fairly certain it was my lack of understanding that caused my issue, thanks for working on the
